test/sagemaker_tests/pytorch/training/integration/__init__.py

```diff
@@ -96,6 +96,32 @@ def get_framework_and_version_from_tag(image_uri):
     return tested_framework, tag_framework_version
 
 
+def get_job_type_from_image(image_uri):
```

Comment:
I think it would be safe to assume that we will only test training DLCs on the `sagemaker_tests/pytorch/training` tests.

If someone is sufficiently motivated to test an image that isn't training type, let's not prevent them from doing what they want to do.